proteus/MeshAdaptPUMI/ErrorResidualMethod.cpp

```diff
@@ -0,0 +1,860 @@
+#include "MeshAdaptPUMI.h"
+#include <PCU.h>
+#include <petscksp.h>
+
+#include <apf.h>
```

Comment:
apf is a library within PUMI that takes care of abstractions like fields and meshes.